src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -150,6 +150,17 @@
           </div>
         </div>
 
+        <!-- Change Password -->
+        <div class="form-group row">
```

Comment:
The label should be displayed this way.

![Bildschirmfoto vom 2019-09-02 16-29-16](https://user-images.githubusercontent.com/1897962/64121417-135f0800-cd9f-11e9-808e-48633fec63f5.png)

```
--- src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html	(date 1565180648000)
+++ src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html	(date 1567434524127)
@@ -152,12 +152,17 @@
 
         <!-- Change Password -->
         <div class="form-group row">
-          <label class="col-sm-3 col-form-label"
-                 i18n>Force Change Password</label>
-          <div class="col-sm-9">
-            <input type="checkbox"
-                   formControlName="force_change_pwd"
-                   value="force_change_pwd">
+          <div class="offset-sm-3 col-sm-9">
+            <div class="custom-control custom-checkbox">
+              <input type="checkbox"
+                     class="custom-control-input"
+                     id="force_change_pwd"
+                     name="force_change_pwd"
+                     formControlName="force_change_pwd">
+              <label class="custom-control-label"
+                     for="force_change_pwd"
+                     i18n>Force Change Password</label>
+            </div>
           </div>
         </div>
```